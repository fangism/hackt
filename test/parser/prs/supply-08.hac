// "supply-08.hac"
// expect-fail

bool x, y;
prs < , what> {
	x => y-
}

