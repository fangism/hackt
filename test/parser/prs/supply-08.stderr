object "what" not found, ERROR!  [5:9..12]
